[FW][FIX]stock_voucher_ux: readme in 17.0

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/stock#574

mav-adhoc committed:
[FIX]stock_voucher_ux: reeamde in 17.0

X-original-commit: af0fc4b7d35d15f1cfbc38916c1868a2ef4d5e62

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/stock/pull/586).

fw-bot-adhoc commented:
@mav-adhoc @jcadhoc cherrypicking of pull request ingadhoc/stock#574 failed.

stdout:
```
CONFLICT (modify/delete): stock_voucher_ux/views/stock_book_views.xml deleted in HEAD and modified in 876a1486 ([FIX]stock_voucher_ux: reeamde in 17.0). Version 876a1486 ([FIX]stock_voucher_ux: reeamde in 17.0) of stock_voucher_ux/views/stock_book_views.xml left in tree.
CONFLICT (modify/delete): stock_voucher_ux/__manifest__.py deleted in HEAD and modified in 876a1486 ([FIX]stock_voucher_ux: reeamde in 17.0). Version 876a1486 ([FIX]stock_voucher_ux: reeamde in 17.0) of stock_voucher_ux/__manifest__.py left in tree.
CONFLICT (modify/delete): stock_voucher_ux/README.rst deleted in HEAD and modified in 876a1486 ([FIX]stock_voucher_ux: reeamde in 17.0). Version 876a1486 ([FIX]stock_voucher_ux: reeamde in 17.0) of stock_voucher_ux/README.rst left in tree.

```

stderr:
```
11:48:08.940770 git.c:444               trace: built-in: git cherry-pick 876a14867ece49ff264b69d07628043e4b17844e
error: could not apply 876a1486... [FIX]stock_voucher_ux: reeamde in 17.0
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port
